schemas/dbt/manifest/v12.json

```diff
@@ -265,6 +265,9 @@
                   "lookback": {
                     "default": 0
                   },
+                  "begin": {
+                    "default": null
+                  },
```

Comment:
It looks like this change was already made in https://github.com/dbt-labs/schemas.getdbt.com, so I'm unsure why this PR is doing it for dbt-core but 🤷